### ISSUE TITLE
Pivot tables, can't rearrange rows or columns in Firefox

### ISSUE THREAD

**Issue by notrom** [metabase/metabase#19973]
**Describe the bug**
When using Firefox browser I can't rearrange fields in the rows or columns of a pivot table. Using Chrome everything seems to work as expected.

**Logs**
JS logs are similar in both browsers. Both have a number of instances of:
```
Warning: can't clean query without metadata!
```
Nothing in the server logs

**To Reproduce**
1. Use Firefox
2. New question > Sample Database > Orders
3. Count by Product -> Category and User -> Source
4. Change to pivot visualisation
5. Settings, drag Category from columns to rows section. *Pivot table does not change*
6. Click "Done"
7. Settings again, note that Category is back in the columns section

**Expected behavior**
Moving fields between rows and columns should rearrange the pivot table and changes should be persisted.

**Screenshots**
Firefox after moving a field:
![image](https://user-images.githubusercontent.com/4504437/151279753-fb671857-bc12-480b-bb43-274230941ea3.png)

Chrome after same steps:
![image](https://user-images.githubusercontent.com/4504437/151279808-99e1b212-0a53-4270-80f1-66e14a192969.png)

**Information about your Metabase Installation:**
```
{
  "browser-info": {
    "language": "en-US",
    "platform": "Win32",
    "userAgent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:96.0) Gecko/20100101 Firefox/96.0",
    "vendor": ""
  },
  "system-info": {
    "file.encoding": "Cp1252",
    "java.runtime.name": "OpenJDK Runtime Environment",
    "java.runtime.version": "13.0.2+8",
    "java.vendor": "Oracle Corporation",
    "java.vendor.url": "https://java.oracle.com/",
    "java.version": "13.0.2",
    "java.vm.name": "OpenJDK 64-Bit Server VM",
    "java.vm.version": "13.0.2+8",
    "os.name": "Windows 10",
    "os.version": "10.0",
    "user.language": "en",
    "user.timezone": "Pacific/Auckland"
  },
  "metabase-info": {
    "databases": [
      "clickhouse",
      "h2",
      "sqlserver"
    ],
    "hosting-env": "unknown",
    "application-database": "postgres",
    "application-database-details": {
      "database": {
        "name": "PostgreSQL",
        "version": "12.1"
      },
      "jdbc-driver": {
        "name": "PostgreSQL JDBC Driver",
        "version": "42.2.23"
      }
    },
    "run-mode": "prod",
    "version": {
      "date": "2022-01-20",
      "tag": "v0.42.0-rc1",
      "branch": "release-x.42.x",
      "hash": "2af85ba"
    },
    "settings": {
      "report-timezone": "UTC"
    }
  }
}
```

**Severity**
Most of our users are on Chrome so can work around this.

**Additional context**
I'm seeing the same behaviour in v0.41.6.

In Chrome when arranging the fields I get a call to http://localhost:3000/api/dataset/pivot in the network tab, this doesn't happen in Firefox.

**Comment by flamber** [comment]
Hi @notrom 
No matter how many times I try, I cannot reproduce this on Firefox 96.0.2 - there were some quirks with 96.0.0
I don't know if this could be caused by an add-on - have you tried in Private/Incognito?
I'm only sitting in Firefox and don't remember ever seeing something like this.

**Comment by notrom** [comment]
@flamber it appears to be an extension issue. Only extension I've got is uBlock Origin. Turned that off for the site and the pivot tables started working as expected. Turned it back off and they stopped rearranging again.

uBlock Origin (v1.40.8) seems to be the culprit. Thanks for prompting me to check that! It's a relatively popular extension and I haven't set many specific rules, my config is pretty vanilla. It's the only thing I've noticed in Metabase that seems to be getting caught up in it.

I'll close this now, thanks for your help.
